pkg/controllers/clusterresourceplacement/controller.go

```diff
@@ -206,15 +206,44 @@ func (r *Reconciler) handleUpdate(ctx context.Context, crp *fleetv1beta1.Cluster
 		klog.V(2).InfoS("Placement has been scheduled", "clusterResourcePlacement", crpKObj, "generation", crp.Generation)
 		r.Recorder.Event(crp, corev1.EventTypeNormal, "PlacementScheduleSuccess", "Successfully scheduled the placement")
 	}
+
+	if r.UseNewConditions {
```

Comment:
>It seems that this is not set to true in this PR. I assume that it's just for dry run?

 The flag can be removed when we finish all the changes in other controllers. Otherwise, cx will see empty conditions in the crp.

>is this PR safe to release?

Yes, as the code won't run during the runtime and it should not matter whether the cluster has crd installed or not.